tests/check_title_syntax.sh

```diff
@@ -19,3 +19,24 @@ if [ ${ERROR} -ne 0 ]; then
 else
     echo "File ending by newlines: OK"
 fi
+
+# Test that there is not more than 5 "=" in title levels
+
+ERROR=0
+
+for entry in $(find $(dirname $0)/../*_* -name '*txt')
+do
+    LINE=$(grep -E "^======+ " ${entry})
```

Comment:
You said "Asciidoc supports 6 "=" for title levels, we should test this to avoid breaking the doc."
A line with 6 = will match here.